clueless/views.py

```diff
@@ -1,8 +1,112 @@
 from django.http import HttpResponse
 from django.template import loader
+from django.contrib.auth.models import User, Player, Character
+import logging
+
+# Get an instance of a logger
+logger = logging.getLogger(__name__)
+
+# HttpResponse functions below here
+
 
 def index(request):
 	#return HttpResponse("Welcome to the world of Clue-Less!")
 	template = loader.get_template('clueless/index.html')
 	context = {}
 	return HttpResponse(template.render(context, request))
+
+
+def startgame(request):
+	"""
+	Creates and starts the game with the specified user_id and character_name
+	"""
+
+	template = loader.get_template('clueless/startgame.html')
+	context = {}
+	response = HttpResponse(template.render(context,request))
+
+	if request.method == 'POST':
+		if'user_id' or 'character_name' not in request.POST:
+			logger.error('user_id or character_name not provided')
+		else:
+			# Gets our expected fields from the user's POST
+			user_id = request.POST('user_id')
+			character_name = request.POST('character_name')
+
+			# Check if our user_id and character_name, add them if they don't
+			user = User.objects.get(id = user_id)
+			character = User.objects.get(character = character_name)
+
+			if user is None:
+				logger.info('user not found, adding them now')
+				# TODO add user to database
+				# TODO create actual user object
+
+			if character is None:
+				logger.info('character not found, adding them now')
+				# TODO add character to database
+				# TODO create actual user object
+
+			# Create a Player object
+			player = Player()
+			player.user = user_id
+			player.character = character_name
+
+			# Constructs our game, saves the changes and starts it
+			game = Game()
+			game.addPlayer(player)
```

Comment:
I did not follow this. So in the Game's startGame function, I'll call the Game's addPlayer function, then in views.py i'll just call Game's startGame? Why call addPlayer myself if startGame is